articles/quickstart/native/ios-swift/interactive.md

```diff
@@ -190,9 +190,9 @@ Press the **Logout** button and verify that:
 :::
 
 :::checkpoint-failure
-Sorry about that. Here's a couple things to double check:
-- Did you enter the correct logout URL value?
-- Make sure the Auth0 Domain and Client ID are set correctly.
+If your application did not launch successfully:
```

Comment:
Same as above: the app launch is not what could fail here. The app will launch successfully.